app/components/supplementary-file-browser.js

```diff
@@ -30,7 +30,7 @@ export default Component.extend(Analytics, {
 
     scrollAnim: '',
     selectedFile: null,
-
+    /* eslint-disable ember/no-function-prototype-extensions */
```

Comment:
It doesn't like this but couldn't find a way around. Suggestions welcome.

Reply:
Oh wow this is some old syntax. https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-function-prototype-extensions.md

old:
```
hasAdditionalFiles: function() {
    return this.get('files.length') > 1;
}.property('files'),
```

new:
```
 hasAdditionalFiles: computed('files', function() {
    return this.get('files.length') > 1;
 }),
```